packages/tokamak/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -240,6 +240,40 @@ contract Deploy is Deployer {
         require(address(uint160(uint256(xdmSenderSlot))) == Constants.DEFAULT_L2_SENDER);
     }
 
+    function upgradeL2OutputOracle(address safeOwner) public {
+        insert("SystemOwnerSafe", safeOwner);
+        deployL2OutputOracle();
+        upgradeL2OutputOracleLosic();
+        sync();
+    }
+
+    function upgradeL2OutputOracleLosic() public broadcast {
```

Comment:
Thank you very much! 😍
Can you check the typo ? (Losic -> Logic) 🙇🏼‍♂️

Reply:
thank you! fixed it!